build_config.rb

```diff
@@ -26,6 +26,7 @@
 
   # ngx_mruby extended class
   conf.gem './mrbgems/ngx_mruby_mrblib'
+  conf.gem :github => 'ksss/mruby-stringio' # required by rack-based-api
```

Comment:
Sorry for my review. Now, we need the following two change:

1. add `conf.gem :github => 'ksss/mruby-stringio'` into `mrbgems/rack-based-api/mrbgem.rake` as rack-based-api dependency and remove the line from `build_config.rb` and `build_config_dynamic.rb`
2. revert this commit https://github.com/matsumoto-r/ngx_mruby/pull/178/commits/f11c764809ca281baef367e142405b78c0bc8de1

I'll merge this PR after pushing the two change.